{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--71b91a75-7aa7-4ba5-ad9e-7125ff6d0b41",
    "created": "2020-05-26T20:30:02.050063Z",
    "modified": "2020-05-26T20:30:02.050063Z",
    "name": "CVE-2019-15892",
    "description": "An issue was discovered in Varnish Cache before 6.0.4 LTS, and 6.1.x and 6.2.x before 6.2.1. An HTTP/1 parsing failure allows a remote attacker to trigger an assert by sending crafted HTTP/1 requests. The assert will cause an automatic restart with a clean cache, which makes it a Denial of Service attack."
}